_posts/en/newsletters/2020-02-05-newsletter.md

```diff
@@ -0,0 +1,279 @@
+slug: 2020-02-05-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter announces the release of Eclair 0.3.3, requests
+help testing a Bitcoin Core maintenance release, links to a new tool for
+experimenting with taproot and tapscript, summarizes discussion about
+safely generating schnorr signatures with precomputed public keys, and
+describes a proposal for interactive construction of LN funding
+transactions.  Also included is our regular section about notable
+changes to popular Bitcoin infrastructure projects.
+
+## Action items
+
+- **Upgrade to Eclair 0.3.3:** this new release includes support for
+  [multipath payments][topic multipath payments], deterministic builds
+  for eclair-core (see the [PR description][eclair1295] below), experimental support
+  for [trampoline payments][topic trampoline payments], and various minor
+  improvements and bug fixes.
+
+- **Help test Bitcoin Core 0.19.1rc1:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+## News
+
+- **Taproot and tapscript experimentation tool:** Karl-Johan Alm
+  [posted][alm btcdeb] to the Bitcoin-Dev list about an experimental branch of his
+  [btcdeb][] tool that supports creating and executing [taproot][topic
+  taproot] and [tapscript][topic tapscript] outputs using a command line
+  tool named `tap`.  For more information, see his detailed
+  [tutorial][tap tutorial].
+
+{% comment %}<!-- Timeline:
+1/26 15:29 gmaxwell
+https://github.com/bitcoin-core/secp258k1/pull/558#discussion_r371027220
+"shouldn't this present an interface that takes the pubkey as an
+argument,"
+
+1/26 16:05 jonasnick
+"Note that in that case we'll need to add the pubkey as extradata to the
+deterministic nonce function, otherwise calling sign with the wrong
+public key may leak the secret key through an (invalid) signature."
+
+1/27 18:36 #secp256k1
+[6:36:02 pm] <gmaxwell> but it's not really a problem in ed25519 because the only thing it does with the public key is stuffs it into the e hash, and presumably it stuffs the same data into the nonce hash?  (if it doesn't they're pants on head stupid and should be shot)
+[...]
+[6:37:49 pm] <gmaxwell> okay the ed25519 people should be shot, it's an input to e and not an input to their nonce function
+[6:38:02 pm] <gmaxwell> (I just checked)
+
+1/28 01:49 https://moderncrypto.org/mail-archive/curves/2020/001012.html
+-->{% endcomment %}
+
+- **Safety concerns related to precomputed public keys used with schnorr signatures:** [libsecp256k1 PR
+  #558][libsecp256k1 #558] proposes adding [BIP340][]-compatible
+  [schnorr signature][topic schnorr signatures] creation and
+  verification to the libsecp256k1 library used by Bitcoin Core and
+  several other Bitcoin programs.  Since BIP340 requires signatures
+  commit to the public key that will be used to validate the signature,
```

Comment:
commit -> to commit/commitments? - not sure what the right word is but missing something here.

Reply:
Changed to commit to